charactersheet/charactersheet/templates/spells.tmpl.html

```diff
@@ -3,14 +3,11 @@
     <table class="table table-responsive table-ac-bordered table-hover">
       <thead>
         <tr>
+          <th></th>
```

Comment:
I think the spells prepared column should still be sort-able. Could we add this functionality back?